<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nombreTienda">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calisto MT&quot;;
border-color: rgb(255, 85, 127);
color: rgb(14, 193, 157);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);
</string>
    </property>
    <property name="text">
     <string>Tienda :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);
</string>
    </property>
    <property name="text">
     <string>Dirección de internet :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);
</string>
    </property>
    <property name="text">
     <string>Dirección física :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Número telefónico :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="direccionITienda">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="direccionFTienda">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numeroTienda">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CrearTienda">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Crear Tienda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AgregarProducto">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModificarProducto">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>590</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Modificar </string>
    </property>
   </widget>
   <widget class="QPushButton" name="EliminarProducto">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>590</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(14, 193, 157);
font: 12pt &quot;Calisto MT&quot;;</string>
    </property>
    <property name="text">
     <string>Eliminar </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Lista de productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Guardar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cargar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaProductos">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>351</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConsultarProducto">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>590</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;Calisto MT&quot;;
color: rgb(14, 193, 157);</string>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
